### ISSUE TITLE
Dev call: changing file permissions in Linux

### ISSUE THREAD

**Issue by hannesdatta** [tilburgsciencehub/website#243]
### Contact Details

h.datta@tilburguniversity.edu

### Is your content request related to a problem you've encountered during your research process? Please describe.

When working w/ Research Cloud / Linux environments, I sometimes check out repos w/o write access to files. But I always forget how to change file permissions in Linux.

### Would you like this to be a Building Block or a Tutorial?

Building Block

### Purpose

Changing file and directory permissions in Linux

We just need this:

https://askubuntu.com/questions/265381/how-to-make-a-directory-permanently-writable

In short,

```
sudo chmod 775 'your directory' -R
```

Maybe also copy the table here for other numbers.

<img width="365" alt="image" src="https://user-images.githubusercontent.com/16557730/138113128-2fb132f7-7ef1-4516-bba8-039ed2be6e45.png">

### Motivation

See above

### Suggested title

How to change file and directory permissions on Linux

### Suggested keywords

research cloud, file permissions, chmod, linux

### Required tools

terminal

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by juliehabets** [comment]
@srosh2000 would you like to take up this content request? Let me know!

**Comment by hannesdatta** [comment]
Still find it super relevant. Let's get someone to do this. @krzysztof99xd @arutyunovv?

**Comment by krzysztof99xd** [comment]
Looks relatively simple and quick. I can do it! However, also some time is needed since there are a lot of new issues which I am considering to take
